### DISCUSSION TITLE
Craftist_0.9

### DISCUSSION THREAD

**Discussion by FlohGro-dev** [FlohGro-dev/Craftist#63] [Announcements]
*DISCLAIMER* the login modal window somehow does not use the dark theme which looks ugly. If you struggle to add the Todoist token just switch Craft to the light theme and it should be readable :)

**fixed:** Support for new Todoist API version

> hopefully this fixes all issues due to the deprecated Todoist API - if not please let me know

<hr /><em>This discussion was created from the release <a href='https://github.com/FlohGro-dev/Craftist/releases/tag/0.9'>Craftist_0.9</a>.</em>

**Comment by justinpocta** [comment]
Not sure if it is the plugin, the latest version of Craft or Ventura, but I just installed after moving back to Todoist recently and experienced a crash when trying to paste into the Token. When I clicked on the input and then either used ⌘-V or trackpad "right click" to bring up the context menu to paste, Craft would crash completely the 3 times I tried pasting. I'm logged in now from typing it out manually, but just figured I'd share in case it was helpful! Thanks! :)

**Reply by FlohGro-dev** [reply]
thanks for the report @justinpocta 

I'm not sure if I can do something against that - glad you were able to get it working by typing it manually!

**Comment by johnccoleman** [comment]
I am having trouble installing the Github file on my Mac - it gives me an option to open with an app or program but none are recognized - basic question I know - but how do I download the file on a Mac after it is in my downloads file?  Thanks

**Reply by FlohGro-dev** [reply]
Did you read the "Installation" section in the readme?

You need to download the .craftx file from the latest release and open it in Craft extensions
